spec/requests/api/v1.0/portfolio_items_spec.rb

```diff
@@ -52,6 +52,10 @@
 
   describe "GET /portfolios/:portfolio_id/portfolio_items" do
     before do
+      portfolio_item_policy_scope = PortfolioItemPolicy::Scope.new(nil, PortfolioItem)
```

Comment:
@eclarizio why do we have to stub these, it can run thru the code as log as RBAC is stubbed.

Reply:
We don't *have* to, but I like being able to separate the concerns and force a specific value, and then we can test the individual cases against the actual `PortfolioItemPolicy::Scope` class.